PCAxis.Web.Controls/VariableSelector/VariableSelectorValueSelect.ascx

```diff
@@ -38,13 +38,11 @@
                         </div>
                     </asp:Panel>
                 </asp:Panel>
-                <asp:Panel runat="server" ID="VariableTitlePanel">
-                    <h3><asp:Label runat="server" ID="VariableTitle" Cssclass="variableselector_valuesselect_variabletitle"/></h3>
+                <asp:Panel runat="server" ID="VariableTitlePanel" CssClass="variableselector_valuesselect_variabletitle">
+                    <asp:Label runat="server" ID="VariableTitle" AssociatedControlID="ValuesListBox" />
+                    <asp:Label runat="server" ID="MandatoryText" CssClass="variableselector_valuesselect_mandatory_text" Visible="False"></asp:Label>
```

Comment:
We should add the star in the CSS so that it is present as it used to be and if someone would like to remove it they are free to to so.

Reply:
The star is now added as CSS